NLPQueue/function.json

```diff
@@ -5,7 +5,7 @@
       "name": "msg",
       "type": "queueTrigger",
       "direction": "in",
-      "queueName": "nlpqueue",
+      "queueName": "%NLP_QUEUE_NAME%",
```

Comment:
This var (`%NLP_QUEUE_NAME%`) is being pulled from local.settings.json (not tracked in Git for security reasons)